packages/web/src/js/Modal.ts

```diff
@@ -1,7 +1,7 @@
 import SelectorEngine from './dom/SelectorEngine';
 import EventHandler from './dom/EventHandler';
 import BaseComponent from './BaseComponent';
-import { getElementFromSelector } from './utils/index';
+import { getElementFromSelector, reflow } from './utils/index';
```

Comment:
Is it necessary to specify `/index` in the path?

Reply:
It is not, fixed.